src/librustc_parse/parser/diagnostics.rs

```diff
@@ -282,6 +286,31 @@ impl<'a> Parser<'a> {
         Err(err)
     }
 
+    fn check_too_many_raw_str_terminators(&mut self, err: &mut DiagnosticBuilder<'_>) -> bool {
+        let prev_token_raw_str = match self.prev_token {
+            Token { kind: TokenKind::Literal(Lit { kind: LitKind::StrRaw(n), .. }), .. } => Some(n),
+            Token {
+                kind: TokenKind::Literal(Lit { kind: LitKind::ByteStrRaw(n), .. }), ..
+            } => Some(n),
```

Comment:
```suggestion
            Token { kind: TokenKind::Literal(Lit {
                kind: LitKind::StrRaw(n) | LitKind::ByteStrRaw(n),
                ..
             }), .. } => Some(n),
```
(the idea... rustfmt probably won't be happy with the formatting of the suggestion ^^)